app/k8s/template.hbs

```diff
@@ -7,44 +7,44 @@
 
   <div class="row">
     <div class="col span-4 text-center option option-primary">
-      <h2>Dashboard</h2>
+      <h2>{{t 'k8sAdvacnedTab.dashboard.title'}}</h2>
```

Comment:
`Advanced` is spelled wrong (here, other places in this file, and in translation files)